partcad/tests/unit/test_import_part.py

```
@@ -7,7 +7,7 @@
 from partcad.actions.part import import_part_action
 
 # Mapping file extensions to formats
-from partcad.shape import EXTENSION_MAPPING
+from partcad.shape import PART_EXTENSION_MAPPING
```

Ruff (0.8.2)

`partcad.shape.PART_EXTENSION_MAPPING` imported but unused

Remove unused import: `partcad.shape.PART_EXTENSION_MAPPING`

(F401)